docs/shared-cluster.md

```diff
@@ -16,24 +16,26 @@ The following diagram is the overview of where our shared cluster lives, and how
 
 ## Authentication
 
-We have the kubeadmin credentials as well as the kubeadmin kubeconfig file. You can use either to authenticate to the cluster.
-
-* Make secrets:
+You can use either to authenticate to the cluster:
 
+* Make secrets, get and set KUBECONFIG, assuming `env` sources `secrets/env`:
```

Comment:
Now that we removed the shared cluster secrets from the secrets SA, I believe we can remove the step to make secrets and source the .env

Reply:
They're not totally removed. We will still have:
- SHARED_CLUSTER_LOCATION
- SHARED_CLUSTER_NAME
- SHARED_CLUSTER_RESOURCE_GROUP_NAME
- SHARED_CLUSTER_CLUSTER_RESOURCE_GROUP_NAME

but they'll rarely change